lib/decidim/vocdoni/version.rb

```diff
@@ -3,10 +3,10 @@
 module Decidim
   # This holds the decidim-meetings version.
   module Vocdoni
-    DECIDIM_VERSION = "~> 0.27.1"
+    DECIDIM_VERSION = "0.27.5"
```

Comment:
```suggestion
    DECIDIM_VERSION = "0.27.5"
    DECIDIM_COMPAT_VERSION = ">= 0.27"
```
Currently, the `DECIDIM_VERSION` is used in the file `decidim-vocdoni.gemspec` to specify the Decidim compatiblity, setting this like this will only make this plugin compatible with Decidim 0.27.5 (and nothing else).
Can you add this additional constant and change the line in that file (btw, add authors there too):
```ruby
Gem::Specification.new do |s|
  s.version = Decidim::Vocdoni.version
  s.authors = ["Andrés Pereira de Lucena", "Anna Topalidi", "Ivan Vergés"]
  s.email = ["andreslucena@gmail.com", "anna@pokecode.net", "ivan@pokecode.net"]
  s.license = "AGPL-3.0"
  s.homepage = "https://github.com/decidim-vocdoni/decidim-module-vocdoni"
  s.required_ruby_version = ">= 3.0"

  s.name = "decidim-vocdoni"
  s.summary = "A decidim vocdoni module"
  s.description = "An elections component for decidim's participatory spaces based on the Vocdoni SDK."

  s.files = Dir["{app,db,config,lib}/**/*", "LICENSE-AGPLv3.txt", "Rakefile", "README.md"]

  s.add_dependency "decidim-core", Decidim::Vocdoni::DECIDIM_COMPAT_VERSION
  s.add_dependency "node-runner", "~> 1.1"
  s.metadata["rubygems_mfa_required"] = "true"
end
```